mitxgraders/helpers/validatorfuncs.py

```diff
@@ -137,6 +170,43 @@ def get_number_of_args(callable_obj):
 
     return num_args
 
+def dict_has_no_collisions(dict_of_iterables):
```

Comment:
This looks nice; did you intend to use it somewhere?

Reply:
Oops. I'll delete it.

It's kind of a long story ... `dict_has_no_collisions` and `all_unique` were copy-pasted from a different branch and accidentally came together.